webdriver-spec.html

```diff
@@ -5602,33 +5605,54 @@
   of <a>trying</a> to <a>get a known element</a>
   with argument <var>element id</var>.
 
+ <li><p><a>Scroll into view</a> the <var>element</var>.
+
  <li><p>Wait in an implementation-specific way up to the <a>session
   implicit wait timeout</a> for <var>element</var> to
   become <a>keyboard-interactable</a>.
 
  <li><p>If <var>element</var> is not <a>keyboard-interactable</a>,
   return <a>error</a> with <a>error code</a> <a>element not interactable</a>.
 
- <li><p><a>Scroll into view</a> the <var>element</var>.
+ <li><p>Let <var>text</var> be the result of <a>getting a property</a>
+    <code>text</code> from the <var>parameters</var> argument.
+
+ <li><p>If <var>text</var> is not a string, return an <a>error</a>
+  with <a>error code</a> <a>invalid argument</a>.
 
- <li><p>If <var>element</var> is not the <a>active element</a>:
+ <li><p>If <var>element</var> is not the <a>active element</a> run
+  the <a>focusing steps</a> for the <var>element</var>.
+
+ <li><p>If the user agent renders <var>element</var> as
+  something other than a text control (for example
+  an <a><code>input</code></a> element in the <a><code>color</code>
+  state</a> being presented as a colorwheel):
 
   <ol>
-   <li><p>Run the <a>focusing steps</a> for the <var>element</var>.
+   <li><p>If <var>element</var> does not have an <a>own property</a>
+    named <code>value</code> return an <a>error</a> with <a>error
+    code</a> <a>element not interactable</a>
+
+   <li><p>If <var>element</var> is not <a>mutable</a> return
+    an <a>error</a> with <a>error code</a> <a>element not
+    interactable</a>.
```

Comment:
We don't have that as an error code in this specification, and the intent seems the same.